app/del_replica.go

```diff
@@ -0,0 +1,38 @@
+package app
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/codegangsta/cli"
+)
+
+func DelReplicaCmd() cli.Command {
+	return cli.Command{
+		Name:      "del-replica",
+		ShortName: "del",
```

Comment:
Does this mean that I type mayactl del-replica or mayactl del ? 

Why not mayactl delete <replica | volume> < replicaname | volumename> ?

Reply:
no this is `longhorn del` and it  is implemented based on their existing implementation of the other commands.